content/influxdb/v2.0/reference/flux/stdlib/slack/endpoint.md

```diff
@@ -39,7 +39,7 @@ If using a Slack webhook, you'll receive a Slack webhook URL when you
 _**Data type:** String_
 
 ### token
-The [Slack API token](https://get.slack.help/hc/en-us/articles/215770388-Create-and-regenerate-API-tokens)
+({{< req >}}) The [Slack API token](https://get.slack.help/hc/en-us/articles/215770388-Create-and-regenerate-API-tokens)
```

Comment:
I don't think this is actually required, seeing as it has a default value. If you're using an old-style Webhook URL, you don't need a token.